### PR TITLE
feat: removed dial string and pdp from modem UI

### PR DESCRIPTION
**Pull request by MMaiero**
> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped

**Screenshots:** If applicable, add screenshots to help explain your solution

**Manual Tests**: Optional description of the tests performed to check correct functioning of changes, useful for an efficient review

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabModemUi.java [hunk 1]
```diff
@@ -22,8 +22,6 @@
 import org.eclipse.kura.web.client.util.MessageUtils;
 import org.eclipse.kura.web.shared.model.GwtModemAuthType;
 import org.eclipse.kura.web.shared.model.GwtModemInterfaceConfig;
-import org.eclipse.kura.web.shared.model.GwtModemPdpEntry;
-import org.eclipse.kura.web.shared.model.GwtModemPdpType;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabModemUi.java:26]: Could we delete also these classes? Are they used somewhere?

**Review by pierantoniomerlino — DISMISSED** [review]

**Review by pierantoniomerlino — DISMISSED** [review]

**Comment by pierantoniomerlino** [comment]
@MMaiero Please update the copyright headers.

**Comment by pierantoniomerlino** [comment]
@MMaiero the `kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtModemInterfaceConfig.java` is still with the old copyright year. Could you please update it?

**Comment by MMaiero** [comment]
> @MMaiero the `kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtModemInterfaceConfig.java` is still with the old copyright year. Could you please update it?

Done

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**